Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scenes/BoundaryVisualization.unity

```diff
@@ -412,6 +412,7 @@ GameObject:
   m_Component:
   - component: {fileID: 353698135}
   - component: {fileID: 353698136}
+  - component: {fileID: 353698137}
```

Comment:
Shouldn't we be updating this on the main dev branch since it's not related to the example scene?

Reply:
I can split it out.

Reply:
Thanks 😄